flake.nix

```diff
@@ -87,6 +87,10 @@
                                 # Uncomment to use a custom database URL
                                 # DATABASE_URL = "postgresql://postgres:...CHANGE-ME";
 
+                                # Uncomment to use a custom session secret, ensuring sessions aren't invalidated
+                                # on each deploy.
```

Comment:
We should propably also add a link to the docs for this one https://ihp.digitallyinduced.com/Guide/deployment.html#ihpsessionsecret

Reply:
Done